jazz-ui/package.json

```diff
@@ -24,46 +24,53 @@
     "@angular/platform-browser": "^2.4.10",
     "@angular/platform-browser-dynamic": "^2.4.10",
     "@angular/router": "^3.3.1",
-    "@ng-bootstrap/ng-bootstrap": "^1.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^1.0.0-alpha.26",
     "@ng-idle/core": "2.0.0-beta.12",
     "@ng-idle/keepalive": "2.0.0-beta.12",
-    "angular2-moment": "^1.8.0",
+    "angular2-moment": "^1.6.0",
     "angular2-number-picker": "^0.8.8",
     "angular2-toaster": "^2.0.0",
     "bootstrap": "^3.3.7",
-    "chart.js": "^2.7.2",
-    "core-js": "^2.5.3",
-    "d3": "^4.13.0",
+    "chart.js": "^2.7.0",
```

Comment:
I don't have the full context here - but what's the reason for downgrading?

Reply:
havent changes this manually, while doing npm update or while merging two repositories this has changed